styles/fonts.js

```diff
@@ -1,5 +1,6 @@
 export default {
```

Comment:
We should have them scale at a normal rate. That way they visually look proportional.